test/core.jl

```diff
@@ -7219,8 +7219,17 @@ end
 struct B33954
     x::Q33954{B33954}
 end
-@test_broken isbitstype(Tuple{B33954})
-@test_broken isbitstype(B33954)
+@test isbitstype(Tuple{B33954})
+@test isbitstype(B33954)
+
+struct A41503{d}
+    e::d
+end
+struct B41503{j,k} <: AbstractArray{A41503{B41503{Any,k}},Any}
+    l::k
+end
+@test !isbitstype(B41503{Any,Any})
+@test_broken isbitstype(B41503{Any,Int})
```

Comment:
Test case from #41359:
```suggestion
@test_broken isbitstype(B41503{Any,Int})

# issue #41349
struct A41349{T}
    x::Ptr{T}
end
struct B41349
    x::A41349{B41349}
end
@test isbitstype(B41349)
@test isbitstype(Tuple{B41349})
```

Reply:
That is already a couple lines higher (where it used to say test_broken)

Reply:
It has another level of nesting for the free type parameter though, so I think it wouldn't hurt to have that as another test case.